### PR TITLE
Use case for velodyne32

### PR DESCRIPTION
**Pull request by HangYin2015**
Add two "latest" use cases for velodyne32:
(1) calibrate.velodyne32: include odsupercomponent, proxy-velodyne32, and odcockpit to viualize point cloud of Velodyne 32 in real time
(2) recording.velodyne32: include odsupercomponent, proxy-velodyne32, and odrecorderh264 to log Velodyne 32 point cloud in headless mode

Both use cases have been successfully tested using odpcapreplay

### CODE REVIEW

File: usecases/latest/velodyne32/calibrate.velodyne32/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+This folder provides the instructions for recording VLP-16 lidar data. A docker-compose file is provided to start all micro-services to decode VLP-16 packets, visualize them as 3D point cloud, and record VLP-16 data. It includes four services: odsupercomponent, odcockpit, opendlv-core-system-proxy-velodyne16 (or proxy-velodyne16 for short), and odrecorderh264. odsupercomponent is used for software component lifecycle management in OpenDaVINCI. odcockpit is a visualization tool of OpenDaVINCI. proxy-velodyne16 listens to VLP-16 packets and decodes them in real time. odrecorderh264 is used for recording. This tutorial assumes that git, Docker, and Docker Compose are installed. To install Docker, follow the tutorial: https://docs.docker.com/engine/installation/linux/ubuntulinux/. In addition, the Docker image of opendlv.core is necessary for running this use case.
```
Comment (chrberger) [inline usecases/latest/velodyne32/calibrate.velodyne32/README.md]: This README.md talks about VLP-16. I think this is a copy-and-paste error?

File: usecases/latest/velodyne32/recording.velodyne32/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+This folder provides the instructions for recording VLP-16 lidar data. A docker-compose file is provided to start all micro-services to decode VLP-16 packets, visualize them as 3D point cloud, and record VLP-16 data. It includes four services: odsupercomponent, odcockpit, opendlv-core-system-proxy-velodyne16 (or proxy-velodyne16 for short), and odrecorderh264. odsupercomponent is used for software component lifecycle management in OpenDaVINCI. odcockpit is a visualization tool of OpenDaVINCI. proxy-velodyne16 listens to VLP-16 packets and decodes them in real time. odrecorderh264 is used for recording. This tutorial assumes that git, Docker, and Docker Compose are installed. To install Docker, follow the tutorial: https://docs.docker.com/engine/installation/linux/ubuntulinux/. In addition, the Docker image of opendlv.core is necessary for running this use case.
```
Comment (chrberger) [inline usecases/latest/velodyne32/recording.velodyne32/README.md]: @HangYin2015 Same here?

**Review by chrberger — CHANGES_REQUESTED** [review]
This looks fine (just two minor things). Did you test this on SnowFox?

**Comment by HangYin2015** [comment]
Thanks for the review! It is a stupid copy-and-paste error. Now both READMEs are updated. The use cases have not been tested on Snowfox yet. I will test it on Snowfox later.

**Review by chrberger — APPROVED** [review]

**Pull request merged**